adapter/config/default_config.go

```diff
@@ -326,6 +326,7 @@ var defaultConfig = &Config{
 		ServiceURLDeprecated: UnassignedAsDeprecated,
 		Username:             "admin",
 		Password:             "$env{cp_admin_pwd}",
+		OrganizationID:       "",
```

Comment:
Let's add this in the config template file too. We can mention that this config is used for integration tests/local tests.